cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss

```diff
@@ -97,6 +97,12 @@ body {
     }
 }
 
+.config-file-preview-modal {
+    .config-content {
+        white-space: pre-wrap;
```

Comment:
Moderate: I think that we need to add `overflow: auto` so we have horizontal scrolling, a few records fit by default but some like http://0.0.0.0:5000/record/7727 would require horizontal scrolling.

Reply:
Have you run webpack? I can't reproduce it.

<details>
<summary>Screenshots of record 7727</summary>
<div>

![image](https://user-images.githubusercontent.com/8863238/107505659-541a9d80-6b9d-11eb-85fe-9275c2653061.png)

![image](https://user-images.githubusercontent.com/8863238/107505677-5a107e80-6b9d-11eb-89f2-bd17cbaa9506.png)

![image](https://user-images.githubusercontent.com/8863238/107505693-60065f80-6b9d-11eb-96ff-f43249951025.png)

</div>
</details>

Reply:
I did, `cernopendata webpack clean buildall`, but still somehow text was coming out of the modal. However, after rebuilding the image it worked, strange. Ready to be merged!